### PR TITLE
task: clean up `dcos task`

### PR DESCRIPTION
**Pull request by tamarrow**
Update `dcos task` only shows active tasks, `--completed` flag to only
show completed tasks and `--all` to show all tasks.

### CODE REVIEW

File: dcos/mesos.py [hunk 1]
```diff
@@ -25,6 +25,13 @@
 logger = util.get_logger(__name__)
 
 
+COMPLETED_TASK_STATES = [
+    "TASK_FINISHED", "TASK_KILLED", "TASK_FAILED", "TASK_LOST", "TASK_ERROR",
+    "TASK_GONE", "TASK_GONE_BY_OPERATOR", "TASK_DROPPED", "TASK_UNREACHABLE",
+    "TASK_UNKNOWN"
+]
```
Comment (tamarrow) [inline dcos/mesos.py]: consistency with UI: https://github.com/dcos/dcos-ui/blob/master/plugins/services/src/js/constants/TaskStates.js

File: cli/dcoscli/task/main.py [hunk 2]
```diff
@@ -29,7 +29,6 @@ def docopt_wrapper(usage, real_usage, **keywords):
     """ A wrapper around the real docopt parser.
     Redirects a failed command to /dev/null and prints the proper
     real_usage message, instead of just the usage string from usage.
-
```
Comment (jimenez) [inline cli/dcoscli/task/main.py]: maybe we want to keep this?

File: dcos/mesos.py [hunk 3]
```diff
@@ -567,7 +584,9 @@ def _framework_dicts(self, inactive=False, completed=False):
                 yield framework
 
         for framework in self.state()['frameworks']:
-            if inactive or framework['active']:
+            if framework['active'] and active:
```
Comment (jimenez) [inline dcos/mesos.py]: what are the other state possibilities here a comment would be great, also couldn't this two conditions fit in one line?

Comment (tamarrow) [inline dcos/mesos.py]: 'active' is a property in the framework dict, that is either True or False. `active` the parameter in the function here, is also a boolean, which means we want to include frameworks that are in the active state. 

I'm not sure what two conditions you are referring to that 'could fit in one line'.

File: cli/dcoscli/task/main.py (hunk 2 above)
Comment (tamarrow) [inline cli/dcoscli/task/main.py]: done

**Pull request merged**